pcsx2-qt/MainWindow.cpp

```diff
@@ -1748,6 +1748,12 @@ void MainWindow::onInputRecNewActionTriggered()
 						m_ui.actionToolbarReset->setEnabled(!g_InputRecording.isTypeSavestate());
 					});
 				}
+				else
+				{
+					Host::ReportFormattedErrorAsync(
+						TRANSLATE_SV("MainWindow", "Input Recording Failed"),
+						TRANSLATE("MainWindow", "Failed to create file: %s"), filePath.c_str());
```

Comment:
```suggestion
					Host::ReportErrorAsync(
						TRANSLATE_SV("MainWindow", "Input Recording Failed"),
						fmt::format(TRANSLATE_FS("MainWindow", "Failed to create file: {}"), filePath));
```

Sorry, should've been more specific. Trying to avoid format strings these days, they're more prone to error.

Reply:
Are you also wanting me to replace the qstring based formatting I have in Input Viewer with fmt? Or just use fmt here

Reply:
Nah, that one's fine, it's not a C format string :)

Reply:
Okay